packages/wrangler/src/inspect.ts

```diff
@@ -399,6 +400,43 @@ export default function useInspector(props: InspectorProps) {
 								params.exceptionDetails.exception?.description ?? ""
 							);
 						}
+
+						const cause =
+							params.exceptionDetails.exception?.preview?.properties.find(
+								(prop) => prop.name === "cause"
+							);
+						if (cause?.value !== undefined) {
```

Comment:
If the "cause" is found will it have a "value" or this just a defensive guard just in case?